src/PAModel/SourceTransforms/AppTestTransform.cs

```diff
@@ -188,7 +188,13 @@ public void BeforeWrite(BlockNode control)
                 // Lookup screenID by Name
                 if (screenProp != null)
                 {
-                    _screenIdToScreenName.ToDictionary(kvp => kvp.Value, kvp => kvp.Key).TryGetValue(screenProp.Expression.Expression, out screenId);
+                    foreach (var prop in _screenIdToScreenName.ToDictionary(kvp => kvp.Value, kvp => kvp.Key))
```

Comment:
why "toDictionary" if you're just going to look at _screenIdToScreenName's KVPs? Seems like you could leave that off entirely & just swap the Key for Value below...